apps/edxapp/templates/cms/_configs/settings.yml.j2

```diff
@@ -6,3 +6,14 @@ SITE_NAME: "{{ project_display_name }}"
 
 LMS_BASE: "{{ edxapp_lms_host }}"
 CMS_BASE: "{{ edxapp_cms_host }}"
+
+# Celery Broker
+# FIXME: we should use the redis_app_* default vars
```

Comment:
I did not understand this sentence?

Reply:
The redis app should be bundled with configuration variables (image, tag, host, port, ...). We will use those variables when they will be available (from your PR).